### DISCUSSION TITLE
Shutting down the application from start_windows.bat not completing via Ctrl+C

### DISCUSSION THREAD

**Discussion by chrisoutwright** [open-webui/open-webui#3467] [General]
# Bug Report

**Bug Summary:**
The start_windows.bat script for startup and then trying to do crtl-c for shutting down the application does not perform any action in the latest version. This functionality worked correctly in previous versions, such as 0.1.124.

**Steps to Reproduce:**
Run the start_windows.bat script on a Windows machine.
Attempt to shut down the application using ctrl-c

**Expected Behavior:**
The application should shut down gracefully when using the start_windows.bat script and then ctl-c.

**Actual Behavior:**
The ctl-c within running script runs but does not perform any action to shut down the application. It just sits there without any output or logs.

## Environment

- Open WebUI Version: v0.3.5
- Ollama (if applicable): 0.1.44
- Operating System: Windows 10

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
No relevant ..
just stuck:
`INFO:     Shutting down`

## Installation Method

Manual

## Additional Information
None

**Comment by AncientMystic** [comment]
Have you tried ctrl+pause/break? 
(Usual suggestion for when ctrl+c fails)

**Comment by chrisoutwright** [comment]
I tried that one as well, but I will get stuck at
INFO: Shutting down
and will have to ungracefully kill it.
